src/include/platform/CHIPDeviceEvent.h

```diff
@@ -441,12 +447,15 @@ struct ChipDeviceEvent final
 
         struct
         {
-            CHIP_ERROR Status;
             uint64_t PeerNodeId;
```

Comment:
PeerFabricIndex is needed here maybe?

Reply:
PeerFabricIndex is not used anywhere right now, we can add it later if there is any scenario we need it.

Reply:
Acked